README.md

```diff
@@ -115,159 +122,149 @@ _See documentation [here](#tensorflow-and-keras)._
-  - [Command Line Interface](#command-line-interface)
-    - [init](#init)
-    - [version](#version)
-    - [experiment](#experiment)
-    - [up](#up)
-    - [down](#down)
-    - [upgrade](#upgrade)
-    - [pull](#pull)
-  - [TensorBoard Experiments](#tensorboard-experiments)
-  - [Anonymized Telemetry](#anonymized-telemetry)
-  - [Contributor Guide](https://github.com/aimhubio/aim/wiki/Contributing)
-
-## Installation
-To install Aim, you need to have python3 and pip3 installed in your environment
-1. Install Aim python package
-```shell
-$ pip install aim
-```
-In order to start Aim UI you need to have Docker installed.
 ```shell
 $ aim up
 ```
+Jump to [[Overview](#overview)] [[SDK Specifications](#sdk-specifications)] [[Use Cases](#use-cases)]
 
-## Concepts
-Aim doesn't use too many concepts that needs explaining. Still, here are a few details to avoid confusion:
-- **Experiment**: it's a grouping mechanism for training runs. Experiments are a way to organize the training runs in a meaningful manner
-- **Run**: it's the single training run researchers start
+# Overview
 
-These two are separated just to enable effective organization of the research work.
+Aim helps you to compare 1000s of training runs at once via versatile SDK and flexible UI.
 
-## Where is the Data Stored 
-When the AI training code is instrumented with [Aim Python Library](#python-library) and ran, aim automatically creates a `.aim` directory where the project is located. All the metadata tracked during training via the Python Library is stored in `.aim`.
-Also see [`aim init`](#init) - an optional and alternative way to initialize aim repository.
+While using Aim SDK you create a Session object. It handles the tracking of metrics and parameters.
 
-## Python Library
-Use Python Library to instrument your training code to record the experiments.
-The instrumentation only takes 2 lines:
-```py
-import aim
-```
-Afterwards, simply use the two following functions to track metrics and any params respectively.
+When the training code is instrumented with Aim SDK's [Python Library](#python-library) and ran, Aim creates the `.aim` repository in your specified path and stores the data. Otherwise the data is created and stored in working directory.
 
-```py
-...
-aim.track(metric_val, name='metric_name', epoch=current_epoch)
-aim.set_params(hyperparam_dict, name='dict_name')
-...
-```
+Additionally, Aim SDK also gives you flexibility to:
+
+- use multiple sessions in one training script to store multiple runs at once. When not initialized explicitly, Aim creates a default session.
+- use experiments to group related runs together. An experiment named `default` is created otherwise.
+- use integrations to automate tracking
+
+Jump to [[Getting Started](#getting-started-in-3-steps)] [[SDK Specifications](#sdk-specifications)] [[Use Cases](#use-cases)]
+
+# SDK Specifications
+
+## Session
+
+Use Session to specify custom `.aim` directory or the experiment from the code.
```

Comment:
Maybe something like:
`Session is the main object that tracks and stores the metadata (metrics and hyperparams).`
`Use Session to specify custom .aim directory, the experiment from the code or other tracking-specific configs from the code`

Reply:
Also add:
`Note: if not created explicitely in your training code, Aim will create a default Session object with default .aim directory and experiment name`

Reply:
note-e hle vor chem avelacnum, cuz verevum vonc vor te ka, mi hat irar het nayenq nor hajordi mej kavelacnenq